schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/index-fields/index.xsl

```diff
@@ -1015,14 +1015,15 @@
         </xsl:apply-templates>
       </xsl:if>
 
+      
       <xsl:variable name="jsonFeatureTypes">[
         <xsl:for-each select="mdb:contentInfo//gfc:FC_FeatureCatalogue/gfc:featureType">{
 
           "typeName" : "<xsl:value-of select="util:escapeForJson(gfc:FC_FeatureType/gfc:typeName/text())"/>",
-          "definition" :"<xsl:value-of select="util:escapeForJson(gfc:FC_FeatureType/gfc:definition/*/text())"/>",
-          "code" :"<xsl:value-of select="util:escapeForJson(gfc:FC_FeatureType/gfc:code/*/text())"/>",
-          "isAbstract" :"<xsl:value-of select="gfc:FC_FeatureType/gfc:isAbstract/*/text()"/>",
-          "aliases" : "<xsl:value-of select="util:escapeForJson(gfc:FC_FeatureType/gfc:aliases/*/text())"/>"
+          "definition" :"<xsl:value-of select="util:escapeForJson(gfc:FC_FeatureType/gfc:definition/gco:CharacterString/text())"/>",
```

Comment:
Should not be `(gco:CharacterString|gcx:Anchor)`?

Reply:
I've never seen `Anchor` used in here. If putting a link to a document for the feature type, it would go in `definitionReference` I suppose

```xml
               <gfc:featureType>
                  <gfc:FC_FeatureType>
                     <gfc:typeName/>
                     <gfc:definition gco:nilReason="missing" xsi:type="lan:PT_FreeText_PropertyType">
                        <gco:CharacterString/>
                     </gfc:definition>
                     <gfc:isAbstract>
                        <gco:Boolean>true</gco:Boolean>
                     </gfc:isAbstract>
                     <gfc:aliases/>
                     <gfc:definitionReference>
                        <gfc:FC_DefinitionReference>
                           <gfc:definitionSource>
                              <gfc:FC_DefinitionSource>
                                 <gfc:source>
                                    <cit:CI_Citation>
                                       <cit:title gco:nilReason="missing" xsi:type="lan:PT_FreeText_PropertyType">
                                          <gco:CharacterString/>
                                          <lan:PT_FreeText>
                                             <lan:textGroup>
                                                <lan:LocalisedCharacterString locale="#FR"/>
                                             </lan:textGroup>
                                             <lan:textGroup>
                                                <lan:LocalisedCharacterString locale="#EN">Data model published by the Sandre</lan:LocalisedCharacterString>
                                             </lan:textGroup>
                                          </lan:PT_FreeText>
                                       </cit:title>
                                       <cit:onlineResource>
                                          <cit:CI_OnlineResource>
                                             <cit:linkage xsi:type="lan:PT_FreeText_PropertyType">
                                                <gco:CharacterString/>
                                                <lan:PT_FreeText>
                                                   <lan:textGroup>
                                                      <lan:LocalisedCharacterString locale="#FR">https://www.sandre.eaufrance.fr/definition/eth/latest</lan:LocalisedCharacterString>
                                                   </lan:textGroup>
```